pkg/disk/cloud.go

```diff
@@ -620,72 +544,35 @@ func tagDiskAsK8sAttached(diskID string, ecsClient *ecs.Client) {
 	klog.Infof("tagDiskAsK8sAttached:: add tag to disk: %s", diskID)
 }
 
-func waitForSharedDiskInStatus(ctx context.Context, retryCount int, interval time.Duration, diskID, nodeID string, expectStatus string, ecsClient *ecs.Client) error {
```

Comment:
Do not delete the logics of sharedisk, try to make batch describe and sharedisk coexists

Reply:
OK. But due to lack of testing, we cannot verify it still works.